engine/updategraph/src/main/java/io/deephaven/engine/updategraph/UpdateGraphAwareCompletableFuture.java

```diff
@@ -140,23 +138,26 @@ public boolean complete(T value) {
     }
 
     public boolean completeExceptionally(Throwable ex) {
-        if (ex == null)
+        if (ex == null) {
```

Comment:
You can use `Objects.requireNonNull` for this. No need to make the change if there's nothing else to change.